data/formats-data.js

```diff
@@ -442,6 +442,15 @@ let BattleFormatsData = {
 		tier: "(PU)",
 		doublesTier: "DUU",
 	},
+	pikachustarter: {
+		eventPokemon: [
+			{"generation": 7, "level": 5, "perfectIVs": 6, "moves": ["thundershock", "tailwhip", "growl"], "pokeball": "pokeball"},
+		],
+		eventOnly: true,
+		isNonstandard: true,
+		isUnreleased: true,
+		tier: "Unreleased",
```

Comment:
It's Invalid in context of Gen7, not Unreleased

Reply:
`Illegal` is I think the text we use?

Reply:
The tier doesn’t matter, it’s the `isNonstandard` tag. Not to mention meltan and melmetal are confirmed unreleased.

Reply:
That wasn’t what I meant.. but alright. I don’t feel comfortable with tiering these as illegal.

Reply:
They're illegal in USUM. This is `data/format-data.js`, which is a USUM file. You can tier them `Unreleased` in `mods/lgpe/format-data.js`, but not in this one.